### ISSUE TITLE
[Question]: How can I create a model with multiple outputs?

### ISSUE THREAD

**Issue by flo22hsma** [SciSharp/TensorFlow.NET#1229]
### Description

My goal is an object detection model with a classifier and a regression for the bounding box.
I have 2 outputs and I need an additional loss function for each. In Python I can compile with a list of 2 losses and in C# I can only pass one loss function as a parameter.
And for the fit method the same for the y value, I can only pass one array instead of a list of two arrays. I have one array for each output.
Does anyone have an idea for my problem?

My current solution is a custom trainingloop.
